logviewer/src/main/java/com/nodesagency/logviewer/data/database/daos/CategoryDao.kt

```diff
@@ -16,6 +16,9 @@ internal interface CategoryDao {
     @Query("SELECT * FROM Categories ORDER BY name ASC")
     fun getAlphabeticallySortedCategories(): DataSource.Factory<Int, Category>
 
+    @Query("SELECT * FROM Categories WHERE (lower(name) LIKE :name)")
```

Comment:
Will this return the proper category if the `:name` is "I/O", but the `name` (the stored value compared) is lowercased to "i/o"?

Reply:
LIKE is supposed to be case insensitive so I've removed lower() completely. Will cover this case with unit tests later